docs/src/app/components/Avatar/ExampleSimple.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Avatar from 'material-ui/lib/avatar';
+import FileFolder from 'material-ui/lib/svg-icons/file/folder';
+import styles from 'material-ui/lib/styles';
+import FontIcon from 'material-ui/lib/font-icon';
+import List from 'material-ui/lib/lists/list';
+import ListItem from 'material-ui/lib/lists/list-item';
+
+const colors = styles.Colors;
+
+const AvatarExampleSimple = React.createClass({
+  render() {
+    return (
+      <List>
+        <ListItem
+          leftAvatar={
+            <Avatar src="images/uxceo-128.jpg" />
+          }
+          disabled={true}
```

Comment:
use `disabled` instead of `disabled={true}`. I think this format looks better:
```js
<ListItem
  disabled
  leftAvatar={
    <Avatar src="images/uxceo-128.jpg" />
  }>
  Image Avatar
</ListItem>